deploy/deploycomponents.yaml

```diff
@@ -0,0 +1,12 @@
+#Base components for setting up a deploy configuration. 
+
+.migrate:
+  script:
+    - cp -r $CI_PROJECT_DIR/sql/* /flyway/sql/
+    - sh /flyway/flyway -url=jdbc:postgresql://$POSTGRES_SERVER:5432/$POSTGRES_DB -schemas=public -user=$POSTGRES_USER -password=$POSTGRES_PASSWORD -placeholders.ENVIRONMENT=$DEPLOYENV migrate
```

Comment:
You swapped to setting `CI_ENVIRONMENT_NAME` (But also, see Sarah's comment about how that is intended to be set by gitlab), but are still using DEPLOYENV here